docs/source/intro.rst

```diff
@@ -0,0 +1,44 @@
+===========
+ome-zarr-py
+===========
+
+Experimental support for multi-resolution images stored in Zarr filesets, according to the `OME zarr spec`_.
+
+
+Features
+--------
+
+- Use as a image reader plugin for `napari`_. See `napari-ome-zarr`_.
+- Simple :doc:`cli` to read and download conforming OME-NGFF filesets.
+- :doc:`python` for reading and writing OME-NGFF filesets.
```

Comment:
These three very important bullet points start each time differently - once with a verb, once with "Simple..." (there is no verb in that sentence) and once even with a link.
Understood this is automatically generated, but it does not read very well.